README.md

```diff
@@ -1,64 +1,201 @@
+   - In the **Supported account types** section, select **Accounts in this organizational directory only ({tenant name})**.
+   - Select **Register** to create the application.
+1. On the application **Overview** page, find the **Application (client) ID** and **Directory (tenant) ID** values and record it for later. You'll need them to configure the values in `ClientCredentialGrant.java` later in your code.
+1. In the Application menu blade, click on the **Certificates & secrets**, in the **Client secrets** section, choose **New client secret**:
+   - Type a key description (for instance `app secret`),
+   - Select a key duration of either **In 1 year**, **In 2 years**, or **Never Expires** as per your security concerns.
+   - The generated key value will be displayed when you click the **Add** button. Copy it for use in the steps later.
+   - You'll need this key later in your code. This key value will not be displayed again and is not retrievable by any other means, so make sure to note it from the Azure portal before navigating to any other screen or blade.
+1. In the Application menu blade, click on the **API permissions** to open the page where we add access to the Apis that your application needs.
+   - Click the **Add a permission** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected
+   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
+   - In the **Application permissions** section, ensure that the right permissions are checked: **User.Read.All**
+   - Select the **Add permissions** button
+
+1. At this stage permissions are assigned correctly but the client app does not allow interaction. Therefore no consent can be presented via a UI and accepted to use the service app.
+   Click the **Grant/revoke admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the requested permissions for all account in the tenant.
+   You need to be an Azure AD tenant admin to do this.
+
+### Step 3:  Configure the sample to use your Azure AD tenant
+
+In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+Open the `ClientCredentialGrant.java` to configure the project.
+
+#### Configure the client project
+
+1. Open the `src\main\java\ClientCredentialGrant` class
+1. Find the line `final static String TENANT_SPECIFIC_AUTHORITY` and replace `Enter_the_Tenant_Info_Here` with your Azure AD **Tenant Id**.
+1. Find the line `final static String CONFIDENTIAL_CLIENT_ID` and replace the existing value with the **Application ID (clientId)** of the `java-daemon-console` application copied from the Azure portal.
+1. Find the line `final static String CONFIDENTIAL_CLIENT_SECRET` and replace the existing value with the **key value** you saved during the creation of the `daemon-console` app, in the Azure portal.
+
+### Step 4: Run the sample
```

Comment:
I think we need to add the steps to run the Java sample similar to other samples

Reply:
Added.